src/util/helpers.ts

```diff
@@ -54,11 +54,16 @@ export function getWebSocketClass(
 /**
  * Given connection metadata, creates a websocket connection string
  */
-export function getConnectionStr(connectionMetadata: GovalMetadata, isPolling: boolean): string {
+export function getConnectionStr(
+  connectionMetadata: GovalMetadata,
+  isPolling: boolean,
+  pollingHost: string | null,
+): string {
   const gurl = urllib.parse(connectionMetadata.gurl);
   if (isPolling) {
-    gurl.hostname = 'gp-v2.herokuapp.com';
-    gurl.host = 'gp-v2.herokuapp.com';
+    const host = pollingHost || 'gp-v2.herokuapp.com';
```

Comment:
```suggestion
    const host = pollingHost ?? 'gp-v2.herokuapp.com';
```

the [nulling coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) is a bit better for missing (rather than falsy) stuff.